tests/test_self_chat.py

```diff
@@ -29,3 +29,16 @@
             ]
         )
         self_chat.self_chat(opt)
+
+    def test_ed(self):
+        pp = self_chat.setup_args()
+        opt = pp.parse_args(
+            [
+                '-mf',
+                'zoo:blender/blender_3B/model',
+                '-t',
+                'empathetic_dialogues',
+                '--seed-messages-from-task',
```

Comment:
Does this not need a true argument?

Reply:
no this is good, it's a `store_true` flag

Reply:
Can we also change that to not be a store_true then? Store_true is impossible to override once it's been set in an opt file

Reply:
Yeah, that's true, I'll change it.

Reply:
Addressed this [here](https://github.com/facebookresearch/ParlAI/pull/3113).